origins_migrations/src/EventSubscriber/PostMigrationSubscriber.php

```diff
@@ -41,7 +41,7 @@ class PostMigrationSubscriber implements EventSubscriberInterface {
   public function __construct(EntityTypeManagerInterface $entity_type_manager,
                               LoggerChannelFactory $logger) {
     $this->entityTypeManager = $entity_type_manager;
-    $this->logger = $logger->get('origins_common');
+    $this->logger = $logger->get('origins');
```

Comment:
I think that you might as well be consistent and log this as 'origins_migrations' rather than 'origins'